apps/taquito-test-dapp/src/Layout.svelte

```diff
@@ -1,6 +1,5 @@
 <script lang="ts">
   import Sidebar from "./lib/Sidebar.svelte";
-  import TaquitoLogo from "./lib/TaquitoLogo.svelte";
```

Comment:
possible to still have a static log somewhere in the background?

Reply:
I think it's OK to remove it for now. I've started a doc to improve the test dApp in notion